LiskHQ/lisk-sdk
# Add faucet UI - Closes #6141

### What was the problem?

This PR resolves #6141 

### How was it solved?

- Implement faucet UI as per design

### How was it tested?

- `npm run start:web` and compare with the design
- Check if invalid address is not possible to input
- Check the error message deign
- Check Responsive design
